### PR TITLE
Add links and blog content for SDS

### PR DESCRIPTION
**Pull request by jpradocueva**
This Pull Request adds blogs and links from the home page to the new blogs.

Please review and feel free to update as you see fit.

> Note, Elizabeth Rose is still working in new blogs that I will add once it has been published.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+ 
+### BAMM CLI
+[Documentation and Download](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/tooling-guide/bamm-cli.html)
+ 
+### Aspect Model Editor (AME)
+<a href="https://github.com/OpenManufacturingPlatform/sds-aspect-model-editor/releases" target="_target"> Releases and Downloads</a> | <a href="https://openmanufacturingplatform.github.io/sds-documentation/ame-guide/4.0.0/introduction.html" target="_target">Documentation</a> | [Blog - Tools & Techniques](sds_blog_tools_techniques.md)
+
+### Whitepapers
+[Semantic Data Structuring](https://github.com/OpenManufacturingPlatform/openmanufacturingplatform.github.io/raw/master/docs/sds/OMP-Semantic-Data-Structuring-Whitepaper.pdf) | [Industry 4.0 Core Information Model](https://github.com/OpenManufacturingPlatform/openmanufacturingplatform.github.io/raw/master/docs/sds/OMP-SDS-Whitepaper_I4.0_Core_Information_Model.pdf), [Blog - Industry 4.0](sds_blog_Industry_4_0.md) | [Product Modeling with BAMM](https://github.com/OpenManufacturingPlatform/openmanufacturingplatform.github.io/raw/master/docs/sds/OMP-SDS-Product-Modeling-Whitepaper.pdf)
```
Comment (atextor) [inline index.md:34]: Replace the `,` after the I40 Core Information Model with ` | `

**Review by atextor — CHANGES_REQUESTED** [review]

**Comment by BirgitBoss** [comment]
Great, looks good. The only thing I miss are the blogs for the whitepapers
For SDS:
https://open-manufacturing.org/blog/2021/09/08/blog-manufacturing-challenges-solved-by-semantic-data-structuring/

I wonder that I do not find a blog for the i4.0 whitepaper...

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
the changes are all ok
see only comment on missing whitepaper blogs

**Comment by jpradocueva** [comment]
@BirgitBoss, I am merging the PR, and then I will create a new PR to address any missing blog.

**Comment by BirgitBoss** [comment]
Yes, this is fine with me

**Pull request merged**